automates/text_reading/src/main/scala/org/clulab/aske/automates/mentions/CrossSentenceMention.scala

```diff
@@ -0,0 +1,105 @@
+package org.clulab.aske.automates.mentions
+
+import org.clulab.odin
+import org.clulab.odin._
+import org.clulab.processors.Document
+import org.clulab.struct.Interval
+
+//same as EventMention but with a different foundBy, no paths, sentence == the sent of the trigger, and a different text method
+//todo: place elsewhere
+//todo: change tokenInterval to something informed and usable
+//TODO: These will be serialized as EventMentions, and then won't be deserialized properly to this subclass.
+class CrossSentenceEventMention(
+                                 labels: Seq[String],
+                                 tokenInterval: Interval,
+                                 trigger: TextBoundMention,
+                                 arguments: Map[String, Seq[Mention]],
+                                 paths: Map[String, Map[Mention, SynPath]],
+                                 sentence: Seq[Int],
+                                 document: Document,
+                                 keep: Boolean,
+                                 foundBy: String,
+                                 attachments: Set[Attachment]
+                               ) extends EventMention(labels,  tokenInterval, trigger, arguments, Map.empty, trigger.sentence, document, keep, foundBy, attachments)
+//
+//  def this(
```

Comment:
Hm, if you use this type of mention, we need to figure out serialization for it before we merge. I can help with that (I can do it or we can meet and talk about it). Also, are there any issues with the span of the mention? I guess if the arg text shows up correctly, the full mention span being somewhat off, that's alright. Let's discuss!

Reply:
As you expected, the arg texts show up correctly, but the full mention span is off. I don't think that causes any major problems (at least for now), but it would be better if we could capture the full mention span correctly as well. Also, I need some help with serializing the mention, so let's definitely talk about this tomorrow. Thank you!

Reply:
I guess this is resolved now, with all you helps! Thanks again, Masha :)